src/main/kotlin/com/atlassian/performance/tools/report/api/distribution/DistributionComparison.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.report.api.distribution
+
+class DistributionComparison(
+    val experimentRelativeChange: Double,
+    val experimentAbsoluteChange: Double,
+    val isExperimentRegressed: Boolean,
+    val isExperimentImproved: Boolean
```

Comment:
The `is` prefix is for the getter only, the field itself should be `val experimentImproved: Boolean`